### PR TITLE
Support for GL_LINE_SMOOTH and GL_POLYGON_SMOOTH

### PR DESCRIPTION
**Pull request by meh**

### CODE REVIEW

**Comment by tomaka** [comment]
Looks good, but you seem to indent with 2 spaces. The rest of the code uses 4 spaces, so this needs to be fixed before I merge.

File: src/ops/draw.rs [hunk 1]
```diff
@@ -79,6 +80,13 @@ pub fn draw<'a, U, V>(context: &Context, framebuffer: Option<&ValidatedAttachmen
     // starting the state changes
     let mut ctxt = context.make_current();
 
+    // check if smoothing is supported, it isn't on OpenGL ES
```
Comment (tomaka) [inline src/ops/draw.rs]: And could you move this inside `sync_smooth`?

**Comment by meh** [comment]
Done.

**Comment by tomaka** [comment]
CI doesn't pass.

**Comment by meh** [comment]
Yeah sorry, I was in a hurry and forgot to even run build :panda_face: 

I'll fix it when I get back home.

**Comment by meh** [comment]
Should be good now.

**Comment by tomaka** [comment]
Thanks \o/

**Pull request merged**